css/main.css

```diff
@@ -0,0 +1 @@
+body { background-color: darkkhaki; }
```

Comment:
Kindly add styles to the project to make it match your chosen design. This milestone includes that.